flopy/utils/util_list.py

```diff
@@ -453,7 +453,7 @@ def get_dataframe(self, squeeze=True):
         dfs = []
         for per in self.data.keys():
             recs = self.data[per]
-            if recs is None or recs is 0:
+            if recs is None:

```

Comment:
@jbellino-usgs you wrote this line (604af67cbbbc2bba7feb97b70a67348f11b9f41a); is this the correct modification? Normally a recarray is tested here.

Reply:
I think it's actually possible that recs == 0 in this case, if I remember correctly.  This means there are no list entries.

Reply:
@langevin-usgs ah, you mean `len(recs) == 0`, which works and have now updated.

(For the record, `recs == 0` also "works", but has a different warning: "FutureWarning: elementwise == comparison failed and returning scalar instead; this will raise an error or perform elementwise comparison in the future.")